block/v3/header.go

```diff
@@ -370,6 +371,16 @@ func (h *Header) SetCrossLinks(newCrossLinks []byte) {
 	h.fields.CrossLinks = append(newCrossLinks[:0:0], newCrossLinks...)
 }
 
+// Slashes ..
+func (h *Header) Slashes() []byte {
+	return append(h.fields.Slashes[:0:0], h.fields.Slashes...)
```

Comment:
what's the difference between v1, v2, and v3?  Any where I can find the document?

Reply:
Don't know about docs, this is @harmony-ek design of versioning the headers and I am following existing structure/convention.